.env.example

```
@@ -51,6 +51,10 @@ GRAFANA_URL=http://grafana:3000
 ANTHROPIC_API_KEY=
 # MCP server endpoint (Streamable HTTP transport, services/mcp-server).
 MCP_SERVER_URL=http://mcp-server:3001/mcp
+# Shared bearer token authenticating ingest to the MCP server's HTTP
+# transport (E20-T3) -- same pattern as ARGOCD_WEBHOOK_TOKEN below.
+# Generate with: python -c "import secrets; print(secrets.token_urlsafe(32))"
+MCP_INTERNAL_TOKEN=change-me-to-a-random-token
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MCP_INTERNAL_TOKEN key should go before the MCP_SERVER_URL key

(UnorderedKey)